bespin_api_v2/urls.py

```diff
@@ -0,0 +1,24 @@
+from data import api as data_api
```

Comment:
If we're going to mix and match v1 and v2 APIs in one `urls.py`, can we import them as `v1_api` and `v2_api`?

I'm also reconsidering this v1/v2 distinction and asking myself what problem it's solving. There don't appear to be any name conflicts between v1 and v2, so I'm wondering if it may have the opposite effect - bringing v1 endpoints into a new v2 route right before discovering we need to break/change them in v2.

Another way to think of it is - what do we gain by mounting existing (v1) API endpoints here under v2?

Reply:
> what do we gain by mounting existing (v1) API endpoints here under v2 ?

Not duplicating the logic before we want to change it. Are you suggesting copying the associated serializers and viewsets from v1(data) into the v2 api.py/serializers.py?